dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -155,6 +155,8 @@ public class DockstoreWebserviceConfiguration extends Configuration {
 
     private boolean localCloudWatchMetrics = false;
 
+    private String autoGeneratedDoiAllowList = null;
```

Comment:
Initially, I was thinking about using this config because I believed that it was commons config and would detect a reload ( https://commons.apache.org/proper/commons-configuration///userguide/howto_reloading.html ) 

Looking more deeply into it, this does not seem to be the case, so we may want a more dynamic location unless we're ok with re-deploying when we want to add organizations .... which isn't too bad with ECS I suppose, so take this as an incomplete thought.

Reply:
My understanding is, and @kathy-t correct me if I'm wrong, is that to update the list at runtime we simply:

1. Change the value in Secrets Manager
2. Force a new deployment in ECS.

But it's still less convenient than an API. I'd thought of doing an API at first, but then switched over to your idea of just a config file setting, as our goal is to eventually get rid of opting out (?).

But if we think there will always be some sort of opt-out, it's probably better to go with an API.

Reply:
Similar to above, wasn't even really thinking API. 

Was thinking
1)  allow-list, configuration like you have for whole organizations. (Non permanent)
2) .dockstore.yml setting for opt-out (permanent)

Reply:
> Similar to above, wasn't even really thinking API.
> 
> Was thinking
> 
>     1. allow-list, configuration like you have for whole organizations. (Non permanent)
> 
>     2. .dockstore.yml setting for opt-out (permanent)

Making sure I understand, are you suggesting:

1. Basically keeping the setting in this PR, incorporating PR feedback.
2. Also adding an opt-out to the .dockstore.yml.

Reply:
Yeah, I think we can live with re-deploying ECS tasks to whitelist orgs since it will be temporary. 
The opt-out (my preference, can discuss) should be in .dockstore.yml and not an api.